docs/guide/robot_sbc/setup_raspberry_pi.md

```diff
@@ -215,6 +216,7 @@ cd projects
 * Get the latest donkeycar from Github.
 
 ```bash
+cd ~
```

Comment:
This is not necessary as you are in the home directory automatically after ssh'ing into the RPi.